docs/pages/usage/namespace_offloading.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Namespace extension
+weight: 3
+---
+### Introduction
+
+The Liqo namespace model extends the K8s namespace concepts introducing the support to remote namespaces, "twins" of the local namespace. 
+Those "twin" namespaces map the home cluster namespace on a remote cluster and its associated resources (e.g., services, configmaps). 
+Regarding pod offloading, remote namespace hosts the offloaded pods belonging to the home twin namespace, as they were executed in the home cluster.
+
+### Quick Offloading
+
+Since the very first version, Liqo provides a label-based mechanism to quickly set-up namespace offloading over remote clusters.
```

Comment:
Would personally avoid *historical* references, as they add no value to the reader and might become soon out-of-date,